### ISSUE TITLE
Fetching bug report file from Slack

### ISSUE THREAD

**Issue by thebiglabasky** [metabase/metabase#50632]
**Context**
For now bug report files are stored in Slack. We need to pull that file and use it to analyze its content to initialize the bug report context, feeding subsequent tasks in the bug reproduction environment creation process.

**Important note: eventually, the file should be stored on our own secure S3 bucket so that we remain agnostic from where the bug gets reported (Slack, Pylon...). This is considered in this issue: #50683**

**Comment by filiphric** [comment]
the title and description of this is confusing to me. fetching a file from Slack is implemented on the debugger already. Fetching files from S3 is indeed something that does not work yet and is blocked by first implementing it

**Comment by thebiglabasky** [comment]
Yeah sorry that one is for the backend part once we'll look into creating repro environment (cf. parent issue) I'll clarify

**Comment by thebiglabasky** [comment]
Closing since this is nowhere to be tackled for now
